### PR TITLE
[WIP] J/DEVOPS-46

### PR DESCRIPTION
**Pull request by baccenfutter**

### CODE REVIEW

File: ies-route53/providers/record.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+def ttl
+  @ttl ||= new_resource.ttl
+end
+
+def overwrite
+  @overwrite ||= new_resource.overwrite
+end
+
+action :create do
+  require 'aws-sdk'
```
Comment (till) [inline ies-route53/providers/record.rb]: @baccenfutter I read somewhere to use `require 'aws-sdk-v1'` to make it more obvious. Can you try this?

Comment (fh) [inline ies-route53/providers/record.rb]: Maybe also add a Exception handler around this with a Chef Error if the gem cant be required.

**Comment by baccenfutter** [comment]
This is the second approach for passing explicit credentials to aws-sdk, which magically chooses to use entirely different credentials for authentication leading to authentication failure. I'd be very happy if you could take a look and solve the riddle.

**Comment by till** [comment]
Closed for #886

**Pull request closed without merging**